frontend/src/Pages/PostDetail.jsx

```diff
@@ -38,14 +38,29 @@ const PostDetail = () => {
           {errMsg}
         </span>
         <h1>{singlePost.title}</h1>
-        <p>{singlePost.body}</p>
+        <div className={classes.grid}>
+          <div>
+            <img
+              src="https://autorhodes.com/sites/default/files/special-offer_4.jpg"
+              alt="test"
+            />
+          </div>
+          <div>Description: {singlePost.body}</div>
+        </div>
         <p>Category: {singlePost.category}</p>
         <p>
           Posted by:{" "}
           <Link to={`/user/${singlePost.user.uuid}`}>
             {singlePost.user.username}
           </Link>
         </p>
+        <p>
+          Last updated on:{" "}
+          <Link to={`/post/${singlePost.updatedAt}`}>
+            {singlePost.updatedAt}
+          </Link>
+          {/* How do I format date information from database? */}
```

Comment:
Hat nichts mit der Database zu tun. Schau mal zum Beispiel hier: https://www.w3docs.com/snippets/javascript/how-to-format-a-javascript-date.html